docs/template-featureRequest.md

```diff
@@ -0,0 +1,10 @@
+**Feature / Enhancement Request Template**
+
+
+**Version:**
+
+
+**Proposed Feature / Enhancement**
```

Comment:
May I suggest two new sections on feature request template:
- **Developer/User-facing?**
- **Existing similar features (if any), and their shortcomings**